packages/angular/.eslintrc.js

```diff
@@ -39,7 +39,22 @@ module.exports = {
     },
     {
       extends: ['@aws-amplify/amplify-ui/jest'],
-      files: ['**/__mocks__/**', '**/__tests__/**'],
+      files: [
+        '**/__mocks__/**',
+        '**/__tests__/**',
```

Comment:
Since these lines don't do match any patterns can remove

Reply:
removed `__mocks__` but actually there are some `__tests__` in the angular package